examples/collateral_auction/config.ts

```diff
@@ -0,0 +1,45 @@
+import { nodeEndpoint, bidder_address } from './const';
+
+export default {
```

Comment:
have this in yaml format so it is user configurable, for nodeEndpoint and bidder_address they can be injected by environment variable

we can expect people know how to edit yaml file but not ts file